source/plugins/go/input/lib/cadvisor.go

```diff
@@ -0,0 +1,1219 @@
+package lib
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"math"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	CADVISOR_SECURE_PORT     = "10250"
+	CADVISOR_NON_SECURE_PORT = "10255"
+	CERT_LOCATION            = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	BEARER_TOKEN_LOCATION    = "/var/run/secrets/kubernetes.io/serviceaccount/token"
```

Comment:
BEARER_TOKEN_FILE will be right naming? and similarly for CERT_LOCATION, it will be CA_CERT_PATH?